content/topics/data-science-specific/datacamp/intro-to-python/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: DataCamp Intro to Python
+ready: true
+topic_needs_review: true
+content_type: topic
+---
+
+Please complete the below course on DataCamp, once you have completed it please ask your facilitator to confirm and move this card.
```

Comment:
"Once you have completed it please ask your facilitator to confirm and move this card" - this should go.

Reason: the cards can be viewed and monitored via Tilde. 
Right now "topic_needs_review" doesn't have any functionality attached to it. I might get it done in time. Maybe not. If not then it will be up to one of the DS crew to make a spreadsheet. In any case, make it simple and people are less likely to drop communication balls.